graphql2/schema.graphql

```diff
@@ -972,7 +973,8 @@ input CalcRotationHandoffTimesInput {
   handoff: ISOTimestamp!
   from: ISOTimestamp
   timeZone: String!
-  shiftLengthHours: Int!
+  shiftLengthHours: Int
```

Comment:
Can we add a deprecation note here? and then we can call the new one `shiftLength` since the type is already clear